app/src/layouts/calendar/actions.vue

```diff
@@ -7,21 +7,16 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
-
-export default defineComponent({
+export default {
```

Comment:
`export default defineComponent({`
Let's be good about this. Same goes for all other instances of this.

Reply:
See https://github.com/hanneskuettner/directus/commit/68ae14da5876d9bb7714f2f40e6e85785f37a8fa.

We'll align those exports in a separate PR.